accepted/010-database.md

```diff
@@ -0,0 +1,41 @@
+# SQLite for Auraed
+
+This feature will accomplish **data persistency after reboots/restarts/failure** because **auraed is a systems daemon built for humans who often make mistakes**.
+
+### Outcomes
+
+  - Allow **persistent records** to exist after a daemon crash, failure.
+  - Allow **persistent records** to exist atomicaly after a gRPC request return. 
+      - There will be a deliberate coupling between the gRPC server and the data layer/model.
```

Comment:
is it a correct assumption that there will be only ever one gRPC server on a given node and therefore this intentional coupling cannot lead to conflicts on the sqlite file?